bin/compile

```diff
@@ -144,7 +144,7 @@ def build_levelzero_jni_lib(rebuild=False):
             [
                 "git",
                 "clone",
-                "https://github.com/beehive-lab/levelzero-jni",
+                "https://github.com/otabuzzman/levelzero-jni#winstall",
```

Comment:
Just to keep a note: We should merge first the dependencies and then update this URL to the official repos.